src/main.ts

```diff
@@ -1,107 +1,23 @@
-import {
-  joinVoiceChannel,
-  createAudioPlayer,
-  entersState,
-  VoiceConnectionStatus,
-} from "@discordjs/voice";
+import { joinVoiceChannel } from "@discordjs/voice";
 
-import { VoiceBasedChannel } from "discord.js";
-import { createDiscordJSAdapter } from "./adapter";
```

Comment:
Could you please remove this adapter.ts file, as we're not going to need this anymore